utils/update_checkout/update-checkout-config.json

```diff
@@ -242,7 +242,7 @@
                 "icu": "release-61-1",
                 "tensorflow": "d1db9860a24af2ce64626fe4c3bee69f83700afa",
                 "tensorflow-swift-bindings": "a7ccb727514414d31df9e403f34fa923bdf6a519",
-                "tensorflow-swift-apis": "16d87eb6aa36bec77570819d8ab00ef71ec3eece"
+                "tensorflow-swift-apis": "use-int-in-tf-apis"
```

Comment:
I think you need to merge swift-apis first?

Reply:
I'm building a toolchain now to verify that `swift build` works with `DeepLearning`.
If it compiles, I will merge the `swift-apis` PR, update the commit hash in this PR, then trigger tests for this PR.